subprojects/logging/src/main/java/org/gradle/internal/logging/events/UpdateNowEvent.java

```diff
@@ -34,6 +34,11 @@ public long getTimestamp() {
     }
 
     @Override
+    public String toString() {
+        return UpdateNowEvent.class.getSimpleName() + " " + timestamp;
```

Comment:
It's for debugging purposes only as far as I am concerned. There are other events in this package that do the same. If we decide to change it then we should do it for all of them.